### PR TITLE
Slaughter Demon

### PR DESCRIPTION
**Pull request by ghost**
Had to recreate because I couldn't resolve the merge conflict.

New side event/antagonist. Simple animal that can use pools of blood as
portals to move around the station. Spooky scary.

-CtrlClick on an adjacent blood pool (or turf with blood on it) to
"enter" (start an infinite duration jaunt
-CtrlClick on a blood pool while jaunted to exit at that pool
-If you're pulling a mob while you enter a pool, the mob gets pulled in
with you, and you devour them, healing yourself
-Night vision

Adds an event (admin only) that spawns one.

Adds wizard item for summoning it. Has objective to kill the wizard (and everyone else).

Also fixes typo/filepath for the scrying orb in the new wizard datum thing.

If someone could help me with mind stuff so I can track how many people it eats that'd be great.

Webm

http://www.singulo.io/tg/src/1433549232359.webm

### CODE REVIEW

File: code/modules/mob/living/simple_animal/slaughter/slaughter.dm [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+
+////////////////////The Powers
+
+/mob/living/simple_animal/slaughter/proc/phaseout(var/obj/effect/decal/cleanable/B)
+	var/turf/mobloc = get_turf(src.loc)
+	var/turf/bloodloc = get_turf(B.loc)
+	if(Adjacent(bloodloc))
+		src.notransform = TRUE
+		spawn(0)
+			src.visible_message("The [src] sinks into the pool of blood, and out of existence.")
```
Comment (Anonus) [inline code/modules/mob/living/simple_animal/slaughter/slaughter.dm]: "and out of existence" sounds really awkward here, it'd work fine with just the first bit.

File: code/modules/mob/living/simple_animal/slaughter/slaughter.dm [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+	origin_tech = "combat=5;biotech=8"
+
+
+
+
+
+
+
+
+//Event
```
Comment (Cheridan) [inline code/modules/mob/living/simple_animal/slaughter/slaughter.dm]: nice whitespace and misplaced comment :^)

Comment (KorPhaeron) [inline code/modules/mob/living/simple_animal/slaughter/slaughter.dm]: fight me Cheri

**Comment by daman997** [comment]
this looks cool :+1:

**Comment by Ausops** [comment]
https://imgur.com/Duv35HS

I did this during a speed-sprite for a telecrystal trade but this may also fit the pull request. If you want to use it just save it and make the file extension .dmi

**Comment by KorPhaeron** [comment]
@Ausops

Love you. Wont be home for a few days to upload it though

**Comment by GunHog** [comment]
It really needs better sounds. There should be ripping and tearing, not the regular 'munching' sound, and the blood enter/exit sound needs to sound more menacing as well.

**Comment by KorPhaeron** [comment]
@GunHog

If you can find some or find someone to do them that'd be great, but I have no experience in making such sounds

**Comment by GunHog** [comment]
@Cuboos is your man. Hunt him down in the IRC.

**Comment by KorPhaeron** [comment]
@GunHog

Will do so, thanks.

**Comment by Cuboos** [comment]
WHAT DO YOU WANT FROM ME?

**Comment by KorPhaeron** [comment]
@Cuboos

Spooky noises to match the sprite Ausops made/the webm I made. Entering/exiting the blood, eating victims, attack noise.

Angry death noise would be cool as well

**Comment by Cuboos** [comment]
https://dl.dropboxusercontent.com/u/9654207/demon_attack1.ogg
https://dl.dropboxusercontent.com/u/9654207/demon_attack2.ogg
https://dl.dropboxusercontent.com/u/9654207/Demon_consume.ogg
https://dl.dropboxusercontent.com/u/9654207/demon_dies.ogg
https://dl.dropboxusercontent.com/u/9654207/enter_blood.ogg
https://dl.dropboxusercontent.com/u/9654207/exit_blood.ogg

will these work?

**Comment by Anonus** [comment]
Those are great.

**Comment by GunHog** [comment]
IT IS AS IF I AM PLAYING DOOM! CUBOOS IS THE MAN.

**Comment by KorPhaeron** [comment]
@Cuboos

Fucking amazing

**Comment by Cuboos** [comment]
I fart a few sounds out in a minute, everyone be all "FUCKING AMAZING"

I spend days making the highest quality sounds within my abilit, everyone be all, "FUCKING AMAZING"

THERE'S NO GREY AREA WITH YOU GUYS.

**Comment by Anonus** [comment]
Maybe you're "FUCKING AMAZING", @Cuboos

**Comment by Cuboos** [comment]
http://i.imgur.com/hSKwH0A.png

**Comment by KorPhaeron** [comment]
Alright I'll be home tommorow to upload the new sprites/sounds, anyone have any last minute suggestions on the name? I really dont much care for "slaughter demon" and I'll probably rename it "Butcher" or something else equally generic unless someone comes up with something better

**Comment by Cuboos** [comment]
Demon of Slaughter.

**Comment by GunHog** [comment]
Name it "Flayed One". http://wh40k.lexicanum.com/wiki/Flayed_One

**Comment by Anonus** [comment]
Slaughter demon sounds pretty good, honestly.

**Comment by KorPhaeron** [comment]
@GunHog

Something like that was in my head when imagining the look of the creature but the sprite Ausops made has has no flesh visible, so the name wouldn't make much sense.

If I started tracking a var for skinned enemies (which I've wanted to do since Wraiths honestly) this PR would probably be shut down for being too edgy anyway

**Comment by Miauw62** [comment]
Slaughter Demon is pretty good imo.

**Comment by KorPhaeron** [comment]
@Miauw62 @Cheridan

Looks like another merge conflict. Before I bother fixing it again can I get approval/denial on this commit? It's been up for 2 weeks now/already had to resolve merge conflicts once, I don't want to maintain it in limbo forever.

**Comment by KorPhaeron** [comment]
I guess @RemieRichards as well because you seemed interested in the idea

**Comment by RemieRichards** [comment]
@KorPhaeron there is a push for more lethality in the game, and we've recently gained a few side antags. I feel this would fit in nicely as another side antag.

**Comment by KorPhaeron** [comment]
@RemieRichards 

I know no mergebegging, but what would the estimatd timeframe on it getting merged be? I don't want to fix merge conflicts only to have it sit another two weeks/conflict again

**Comment by RemieRichards** [comment]
@KorPhaeron fix the conflict and ping me (@RemieRichards) tomorrow (fix the conflict tomorrow too if you want to wait) sometime around 5 GMT would mean I'd see it as soon as I got home.

**Comment by KorPhaeron** [comment]
@RemieRichards 

Alright, thanks

**Comment by Miauw62** [comment]
If RR approves of this PR, I can merge this on her behalf as soon as you fix the conflicts.

**Comment by PKPenguin321** [comment]
????? closed why

**Pull request closed without merging**